mods/ra/maps/red-zone/map.yaml

```diff
@@ -0,0 +1,3352 @@
+MapFormat: 11
+
+RequiresMod: ra
+
+Title: redZoneTest
+
+Author: American Blunt
+
+Tileset: DESERT
+
+MapSize: 132,132
+
+Bounds: 1,1,129,129
+
+Visibility: MissionSelector
+
+Categories: Campaign
+
+LockPreview: True
+
+Players:
+	PlayerReference@Neutral:
+		Name: Neutral
+		OwnsWorld: True
+		NonCombatant: True
+		Faction: england
+	PlayerReference@England Missile Silos:
+		Name: England Missile Silos
+		NonCombatant: True
+		Faction: allies
+		LockColor: True
+		Color: ABB7E4
+		Allies: England Northern Base, England Main Base, England Air Base, England Prison Base
+		Enemies: Creeps, USSR, BadGuy
+	PlayerReference@England Air Base:
+		Name: England Air Base
+		NonCombatant: True
+		Faction: allies
+		LockColor: True
+		Color: ABB7E4
+		Allies: England Northern Base, England Main Base, England Prison Base, England Missile Silos
+		Enemies: Creeps, USSR, BadGuy
+	PlayerReference@England Northern Base:
+		Name: England Northern Base
+		NonCombatant: True
+		Faction: allies
+		LockColor: True
+		Color: ABB7E4
+		Allies: England Main Base, England Air Base, England Prison Base, England Missile Silos
+		Enemies: Creeps, USSR, BadGuy
+	PlayerReference@England Prison Base:
+		Name: England Prison Base
+		NonCombatant: True
+		Faction: allies
+		LockColor: True
+		Color: ABB7E4
+		Allies: England Northern Base, England Main Base, England Air Base, England Missile Silos
+		Enemies: Creeps, USSR, BadGuy
+	PlayerReference@England Main Base:
+		Name: England Main Base
+		NonCombatant: True
+		Faction: allies
+		LockColor: True
+		Color: ABB7E4
+		Allies: England Northern Base, England Air Base, England Prison Base, England Missile Silos
+		Enemies: Creeps, USSR, BadGuy
+	PlayerReference@Creeps:
+		Name: Creeps
+		NonCombatant: True
+		Faction: england
+		Enemies: USSR
+	PlayerReference@USSR:
+		Name: USSR
+		Playable: True
+		LockFaction: True
+		Faction: soviet
+		LockColor: True
+		Color: FE1100
+		LockSpawn: True
+		LockTeam: True
+		Allies: BadGuy
+		Enemies: Creeps, England Missile Silos, England Main Base, England Northern Base, England Air Base, England Prison Base
+	PlayerReference@BadGuy:
+		Name: BadGuy
+		NonCombatant: True
+		Faction: soviet
+		LockColor: True
+		Color: FE1100
+		Allies: USSR
+		Enemies: Creeps, England Missile Silos, England Prison Base, England Main Base, England Northern Base, England Air Base
+
+Actors:
+	Actor1243: cycl
```

Comment:
This is a pain, but all fences and walls need to be assigned a correct owner for targeting purposes. (England I assume)